react-devtools report Error: Cannot find module './app'

react-devtools version: 4.22.0
npm -g install react-devtools
react-devtools

```
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module './app'
Require stack:
- /Users/foo/.nvm/versions/node/v14.18.1/lib/node_modules/react-devtools/bin.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.resolve (internal/modules/cjs/helpers.js:99:19)
    at Object.<anonymous> (/Users/foo/.nvm/versions/node/v14.18.1/lib/node_modules/react-devtools/bin.js:32:46)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/foo/.nvm/versions/node/v14.18.1/lib/node_modules/react-devtools/bin.js'
  ]
}
```

--- a/packages/react-devtools/package.json
+++ b/packages/react-devtools/package.json
@@ -11,7 +11,13 @@
   "bin": {
     "react-devtools": "./bin.js"
   },
-  "files": [],
+  "files": [
+    "bin.js",
+    "app.html",
+    "app.js",
+    "index.js",
+    "icons"
+  ],
   "scripts": {
     "start": "node bin.js"
   },

